onnx/version_converter/convert.h

```diff
@@ -564,8 +564,14 @@ class DefaultVersionConverter : public BaseVersionConverter {
     registerAdapter(std::make_unique<CompatibleAdapter>("Size", OpSetID(18), OpSetID(19)));
 
     /******** 19 -> 20 ********/
-    registerAdapter(std::make_unique<GridSample_19_20>());
     registerAdapter(std::make_unique<CompatibleAdapter>("ConstantOfShape", OpSetID(19), OpSetID(20)));
+    registerAdapter(std::make_unique<CompatibleAdapter>("ReduceMax", OpSetID(19), OpSetID(20)));
+    registerAdapter(std::make_unique<CompatibleAdapter>("ReduceMin", OpSetID(19), OpSetID(20)));
+    registerAdapter(std::make_unique<GridSample_19_20>());
+
+    /******** 20 -> 19 ********/
+    registerAdapter(std::make_unique<CompatibleAdapter>("ReduceMax", OpSetID(20), OpSetID(19)));
```

Comment:
This isn't correct. It should fail if the type is boolean.

Reply:
Something like [this](https://github.com/onnx/onnx/blob/262ed7e6611a3161097e1dad488063b3c4b932f7/onnx/version_converter/convert.h#L207) can be used. It needs to be cleaned up a bit (I notice that its error message is specific to "Add" op).